pkg/podvolume/backupper.go

```diff
@@ -337,7 +367,8 @@ func (b *backupper) WaitAllPodVolumesProcessed(log logrus.FieldLogger) []*velero
 	case <-b.ctx.Done():
 		log.Error("timed out waiting for all PodVolumeBackups to complete")
 	case <-done:
-		for _, pvb := range b.result {
+		for _, obj := range b.pvbIndexer.List() {
+			pvb := obj.(*velerov1api.PodVolumeBackup)
```

Comment:
Is there a risk for panic?  Maybe we should be a little more defensive to log an error and continue?